src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OtherPartyOption.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Value
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OtherPartyOption {
+
+    OtherPartyOptionDetails value;
+
+    @JsonCreator
+    public OtherPartyOption(@JsonProperty("value") OtherPartyOptionDetails value) {
```

Comment:
Consider using CcdValue instead of introducing another class

Reply:
Not sure what you meant here, I am following the pattern Jack used on OtherPartyHearingRecordingReq